canonical/ubuntu.com
# Intel NUC desktop rebrand

## QA

- Check https://ubuntu-com-13570.demos.haus/download/intel-nuc-desktop against prod/[copy docs](https://docs.google.com/document/d/1hDkrBqpJXWaENYZMSxd8P-XLFQ2LQqS1wNe4o2MGLho/edit#heading=h.r96oqmuwezfa)
- [Visual design] Check against screenshot below (it's a template, to be used as reference)

## Issue / Card

Fixes https://ubuntu-com-13570.demos.haus/

## Screenshots

![managed](https://github.com/canonical/ubuntu.com/assets/14939793/e16cebbb-32b2-4d6c-9e91-2fb52c95600f)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
